lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -230,16 +233,15 @@ def sea_mask(landmask, orog_band):
                 standard_orography.data > thresholds[0]).astype(int)
             mask_data = sea_mask(standard_landmask.data, orog_band)
             mask_cube = _make_mask_cube(mask_data, key, coords,
-                                        lower_threshold=thresholds)
+                                        topographic_bounds=thresholds)
         elif key == 'land':  # regular topographical bands above land
```

Comment:
I think that the `key == 'max_land_threshold'` block might not be necessary now, as we've taken out the functionality to only specify a single bound, in favour of mandating that both an upper and lower bound need to be specified.